assets/style.css

```diff
@@ -0,0 +1,998 @@
+/* General definitions */
+
+@font-face {
+  font-family: "Roboto Condensed";
+  src: url(fonts/RobotoCondensed-Regular.ttf);
+}
+
+html * {
+  font-family: "Roboto Condensed", arial, sans-serif;
+}
+
+body {
+  background-color: #f4f5f6;
+  min-width: 450px;
+}
+
+.container,
+.container-fluid {
+  max-width: 1310px;
+}
+
+.bg-primary {
+  background-color: #f72210 !important;
+}
+
+.text-primary {
+  color: #f72210 !important;
+}
+
+.border *,
+.border {
+  border: solid 2px black !important;
+}
+
+.border-none {
+  border: none !important;
+}
+
+/* links */
+
+a:hover,
+a:link,
+a:active,
+a:visited {
+  color: currentcolor;
+  text-decoration: none;
+}
+
+.link-c-red a:hover,
+.link-c-red:hover {
+  color: #f72210;
+}
+
+.link-c-red2 a:hover,
+.link-c-red2:hover {
+  color: #f92859;
+}
+
+.link-c-black a:hover,
+.link-c-black:hover {
+  color: black !important;
+}
+
+.link-u-red a:hover,
+.link-u-red:hover {
+  text-decoration: underline red !important;
+}
+
+.link-b-gray a:hover,
+.link-b-gray:hover {
+  background-color: #dbdcdd;
+}
+
+/* main */
+
+main {
+  max-width: 1310px;
+}
+
+@media (min-width: 992px) {
+  .bg-lg-white {
+    background-color: white !important;
+  }
```

Comment:
If you have used the bg-white class here you should not need to override it using important

Reply:
Well, as there's actually no definition for `bg-lg-white`, `bg-md-white` and `bg-sm-white`,  I was trying to do my personal definitions to override the `bg-` defininitions, but  as they use the `!important`  value in the bootstrap definition then I need to use the `!important` values too, otherwise the new definitions won't override the old ones.